addons/account_edi/models/account_edi_format.py

```diff
@@ -249,7 +249,7 @@ def _decode_pdf(self, filename, content):
             pdf_reader = OdooPdfFileReader(buffer, strict=False)
         except Exception as e:
             # Malformed pdf
-            _logger.exception("Error when reading the pdf: %s" % e)
+            _logger.warning("Error when reading the pdf: %s", e)
```

Comment:
```suggestion
            _logger.warning("Error when reading the pdf: %s", e, exc_info=True)
```

This way we have more information in logs